usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -171,7 +170,6 @@ def get_download_ids(keyword, field, size=10000):
     total = get_total_results(keyword, '*', max_iterations)
     if not total:
         logger.error('Error retrieving total results. Max number of attempts reached')
-        return None
```

Comment:
What is the behavior you desire by not returning here? Wouldn't line 174 be a python exception if total is `None`?